src/index.js

```diff
@@ -1,6 +1,6 @@
 import './index.css';
 import toggleIconPrimary from '../assets/toggleIcon.svg';
-import toggleIconSecundary from '../assets/toggleIconSecundary.svg';
+import toggleIconSecondary from '../assets/toggleIconSecondary.svg';
```

Comment:
@juliansg022 What do you think if you named the icons 'toggleIconOpen' and 'toggleIconClose' or something similar?, it could be more legible.